Openshift-EE/pipelines/GitLab/stages/4-chaos/N2QR-cstor-pool-delete/cstor-pool-delete

```diff
@@ -37,7 +37,7 @@ sed -i -e 's/value: percona-mysql-claim/value: repo-data-gitlab-openshift-gitaly
 -e 's/name: openebs-pool-failure/name: openebs-pool-delete/g' \
 -e 's/value: '\''name=percona'\''/value: '\''app=gitaly'\''/g' \
 -e 's/pool-kill/pool-delete/g' \
--e 's/enable//g' \
+-e 's/enable/disable/g' \
```

Comment:
@shashank855  can you check the litmusbook condition once before doing any replace for data persistency.